web/.idea/integreat-app-web.iml

```diff
@@ -2,6 +2,7 @@
 <module type="WEB_MODULE" version="4">
   <component name="NewModuleRootManager">
     <content url="file://$MODULE_DIR$/..">
+      <excludeFolder url="file://$MODULE_DIR$/../.idea/shelf" />
```

Comment:
why did you exclude this folder? Do you shelf files often?

Reply:
Because intelliJ told me there are ignored files that are not excluded :D But I think this is something that is personal and should not be part of the repo?

Reply:
yes think so, too